content/pt/docs/reference/access-authn-authz/authentication.md

```diff
@@ -0,0 +1,892 @@
+---
+reviewers:
+- femrtnz
+- jcjesus
+title: Autenticação
+content_type: concept
+weight: 10
+---
+ 
+<!-- overview -->
+Essa página demonstra uma visão geral sobre autenticação
+ 
+<!-- body -->
+## Usuários em Kubernetes
+ 
+Todos os clusters Kubernetes possuem duas categorias de usuários: contas de serviço gerenciadas por Kubernetes e usuários normais.
+ 
+Assume-se que um serviço independente do cluster gerencia usuários normais das seguintes formas:
+ 
+- Um administrador distribuindo chaves privadas
+- Uma base de usuários como Keystone ou Google Accounts
+- Um arquivo com uma lista de nomes de usuários e senhas
+ 
+Neste quesito, _Kubernetes não possui objetos que possam representar as contas de um usuário normal._ Usuários normais não podem ser adicionados ao _cluster_ através de uma chamada para a API.
+ 
+Apesar de um usuário normal não poder ser adicionado através de uma chamada para a API, qualquer usuário que apresente um certificado válido e assinado pela autoridade de certificados (CA) do _cluster_ é considerado autenticado. Nesta configuração, Kubernetes determina o nome do usuário baseado no campo de nome comum no sujeito (_subject_) do certificado (por exemplo: "/CN=bob"). A partir daí, o subsistema de controle de acesso baseado em função (RBAC) determina se o usuário é autorizado a realizar uma operação específica sobre o recurso. Para mais detalhes, veja a referência sobre o tópico de usuários normais dentro de [requisição de certificado](/docs/reference/access-authn-authz/certificate-signing-requests/#normal-user).
+ 
+Em contraste a usuários normais, contas de serviço são considerados usuários gerenciados pela API do Kubernetes. Elas estão vinculadas à específicos _namespaces_ e criados automaticamente pelo servidor de API ou manualmente através de chamadas da API. Contas de serviço estão ligadas a um conjunto de credenciais armazenados como `Secrets`, aos quais são montados dentro dos _pods_ assim permitindo que processos internos ao _cluster_  comuniquem-se com a API do Kubernetes.
+ 
+Requisições para a API estão ligadas a um usuário normal, conta de serviço ou serão tratadas como requisições anônimas (#anonymous-requests). Isto significa que cada processo dentro ou fora do _cluster_, desde um usuário humano entrando `kubectl` de uma estação de trabalho, a `kubelets` rodando nos nós, a membros do painel de controle devem autenticar-se ao realizarem suas requisições para o servidor API ou serão tratados como usuário anônimo.
```

Comment:
`requisições anônimas (#anonymous-requests)` to `[requisições anônimas](#anonymous-requests)`